model/cicd/spans.yaml

```diff
@@ -0,0 +1,26 @@
+groups:
+  - id: span.cicd.pipeline.run.server
+    type: span
```

Comment:
please add entity/resource associations similar to https://github.com/open-telemetry/semantic-conventions/pull/2123 for the resources you expect to be populated

Reply:
I added entity associations in 7f81a46a421b6a245446f68bab7b7c874dde7e43.